config/default/kustomization.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../crd
-- ../rbac
-- ../manager
-- ../webhook
+  - ../crd
+  - ../rbac
+  - ../manager
+  - ../webhook
```

Comment:
Can we do something so the indentation isn't changed every PR :D

Reply:
I think this is mainly done by the YAML extension I use by RedHat, see: https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml.

Does everyone have that installed?